front/src/applications/operationalStudies/components/MacroEditor/NGE.tsx

```diff
@@ -42,9 +48,15 @@ const NGE = ({ dto }: { dto?: NetzgrafikDto }) => {
       setNgeRootElement(ngeRoot);
 
       // listens to create, update and delete operations
-      // ngeRoot.addEventListener('operation', (event: Event) => {
-      //   console.log('Operation received', (event as CustomEvent).detail);
-      // });
+      ngeRoot.addEventListener('operation', async (event: Event) => {
+        const customEvent = event as CustomEvent;
+        const op = customEvent.detail as NGEEvent;
+        if (onOperation) onOperation(op);
+      });
+
+      if (dto) {
+        ngeRoot.netzgrafikDto = dto;
+      }
```

Comment:
This should be removed: it was part of the old logic, but is now handled by the second `useEffect` below.

(Probably remnant from a rebase.)

Reply:
Done